DANS-KNAW/easy-deposit-report
# EASY-1368: easy-deposit-report

Fixes EASY-1368

- [x] Implement all funcionality.

#### When applied it will
* create a full (or summary) report for all depositors in easy-ingest-flow and easy-sword2 if the depositor option is not used
* create a full (or summary) report for a depositor in easy-ingest-flow and easy-sword2 if the depositor option is used

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?
Apply run-reset-env.sh from the command line to create home directory after cloning the project.
After run-reset-env.sh, create two directories named easy-ingest-flow-inbox and easy-sword2 and copy the related data instances to these two directories.
Run the project with run.sh summary, run.sh full to see all depositors, and with run.sh summary depositorName, run.sh full depositorName for one depositor.


#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
